aws-cpp-sdk-core/source/http/windows/WinHttpSyncHttpClient.cpp

```diff
@@ -133,6 +125,15 @@ void* WinHttpSyncHttpClient::OpenRequest(const Aws::Http::HttpRequest& request,
     HINTERNET hHttpRequest = WinHttpOpenRequest(connection, StringUtils::ToWString(HttpMethodMapper::GetNameForHttpMethod(request.GetMethod())).c_str(),
         wss.c_str(), nullptr, nullptr, accept, requestFlags);
 
+    if (!m_verifySSL)
```

Comment:
To be honest, I am not an expert on the topic. But...
Take a look here https://msdn.microsoft.com/en-us/library/windows/desktop/aa384066(v=vs.85).aspx - the big table almost at the bottom of the page. According to it, WINHTTP_OPTION_SECURITY_FLAGS does not work on session handles. It was confirmed by the fact the old code didn't work. Every time I tested it, it reached this row 'AWS_LOG_FATAL(GetLogTag(), "Failed to turn ssl cert ca verification off.");' and the verifySSL flag had no effect at all.
On the other hand, the proposed code change works just fine and according to Microsoft's documentation it makes sense to be written this way.